enterprise/backend/test/metabase_enterprise/data_editing/notification_test.clj

```diff
@@ -63,19 +64,23 @@
                               email
                               #"Crowberto Corv has created a row for CATEGORIES"
                               #"NAME: New Category"))))
-    :channel/http    (fn [[req :as reqs]]
-                       (is (= 1 (count reqs)))
-                       (is (=? {:body {:event_info {:actor       {:common_name "Crowberto Corv"
-                                                                  :email "crowberto@metabase.com"
-                                                                  :first_name "Crowberto"
-                                                                  :last_name "Corv"}
-                                                    :actor_id    (mt/user->id :crowberto),
-                                                    :created_row {"ID" (mt/malli=? :int) "NAME" "New Category"}
-                                                    :table       {:name "CATEGORIES"}
-                                                    :table_id    (mt/id :categories)}
-                                       :event_name :event/data-editing-row-create,
-                                       :type "system_event"}}
-                               req)))}))
+    :channel/http    (fn [reqs]
+                       (let [reqs (filter (comp #{"row"} namespace :action :event_info :body) reqs)]
```

Comment:
Need to stop the other actions triggering the channels. Should do this in a general way rather than all my hacks.